default/savedsearches.conf

```diff
@@ -2,8 +2,8 @@
 disabled = 0
 enableSched = 1
 alert.track = 0
-cron_schedule = 0 3 * * 0
-description = Runs every week and updates the Max-Mind database.
+cron_schedule = 1 7 * * *
```

Comment:
schedule should be "0 7 * * *" for the sharp 7 o'clock.

Reply:
To keep less conflict I have moved to 7:01